tests/integrations/gcp/test_gcp.py

```diff
@@ -48,21 +48,30 @@ def event_processor(event):
     time.sleep(1)
     return event
 
+def envelope_processor(envelope):
+    (item,) = envelope.items
+    return item.get_bytes()
+
 class TestTransport(HttpTransport):
     def _send_event(self, event):
         event = event_processor(event)
         # Writing a single string to stdout holds the GIL (seems like) and
         # therefore cannot be interleaved with other threads. This is why we
         # explicitly add a newline at the end even though `print` would provide
         # us one.
-        print("EVENTS: {}".format(json.dumps(event)))
+        print("EVENT: {}".format(json.dumps(event)))
+
+    def _send_envelope(self, envelope):
+        envelope = envelope_processor(envelope)
+        print("\\nENVELOPE: {}\\n".format(envelope.decode(\"utf-8\")))
 
-def init_sdk(timeout_warning=False, **extra_init_args):
+def init_sdk(timeout_warning=False, traces_sample_rate=0, **extra_init_args):
     sentry_sdk.init(
         dsn="https://123abc@example.com/123",
         transport=TestTransport,
         integrations=[GcpIntegration(timeout_warning=timeout_warning)],
         shutdown_timeout=10,
+        traces_sample_rate=traces_sample_rate,
```

Comment:
like in aws I don't think this is necessary because of `**extra_init_args`

Reply:
Made this change, this parameter was redundant.